vignettes/internal-impact.Rmd

```diff
@@ -207,56 +209,51 @@ str(old_impact)
 and we can see from a plot that the output is very similar
 
 
-```r
-impact_plot <- merge(old_impact, impact, all.x = TRUE, all.y = TRUE,
-                     by = c("country", "vaccine", "activity_type", "year",
-                            "burden_outcome"))
-countries <- unique(impact_plot$country)
-impact_plot <- impact_plot %>%
-  dplyr::rename(recipe_impact = impact.x, function_impact = impact.y) %>%
-  tidyr::pivot_wider(names_from = country,
-                     values_from = c(recipe_impact, function_impact))
-impact_plot <- as.data.frame(impact_plot)
-countries_filter <- lapply(unique(countries), function(country) {
-  button <- list(
-    method = "restyle",
-    args = list("y", c(
-      list(impact_plot[, paste0("recipe_impact_", country)]),
-      list(impact_plot[, paste0("function_impact_", country)]))),
-    label = country
-  )
-})
-plot <- plotly::plot_ly(data = impact_plot) %>%
-  plotly::add_trace(x = ~year, y = ~recipe_impact_AFG, name = "recipe",
-                    type = "scatter", mode = "markers", text = ~vaccine,
-                    marker = list(
-                      color = "rgb(235, 204, 42)",
-                      size = 10
-                    )) %>%
-  plotly::add_trace(x = ~year, y = ~function_impact_AFG, name = "function",
-                    type = "scatter", mode = "markers", text = ~vaccine,
-                    marker = list(
-                      color = "rgb(60, 154, 178)",
-                      symbol = "circle-open",
-                      size = 10,
-                      line = list(
-                        color = "rgb(60, 154, 178)",
-                        width = 2
-                      )
-                    )) %>%
-  plotly::layout(
-    yaxis = list(title = "impact"),
-    updatemenus = list(
-      list(
-        y = 0.7,
-        buttons = countries_filter
-      )
-    ))
-```
+
+
+
+
+
+
+
+
+
+
+
 
 
 
  
+### HepB 
+```{r} 
+htmltools::tags$iframe( 
+  src = "impact_comparisons_hepb.html", 
+  width = "100%", 
+  height = "400", 
+  scrolling = "no", 
+  seamless = "seamless", 
+  frameBorder = "0", 
+  `data-external` = "1" 
+)
+```
+
+### Measles 
+```{r} 
+htmltools::tags$iframe( 
+  src = "impact_comparisons_measles.html", 
+  width = "100%", 
+  height = "400", 
+  scrolling = "no", 
+  seamless = "seamless", 
+  frameBorder = "0", 
+  `data-external` = "1" 
+)
+```
+
+### Comparisons of HepB, Measles <!-- WIDGET --> YF 
+This shows bars of mean difference between old impact <!-- WIDGET --> new impact method for each 
```

Comment:
Something up here with `<!-- WIDGET -->` turning up twice

Reply:
Thanks, was failing to escape the & in the sed replacement string. Fixed this now